pkgs/misc/vim-plugins/generated.nix

```diff
@@ -2517,6 +2517,18 @@ final: prev:
     meta.homepage = "https://github.com/edwinb/idris2-vim/";
   };
 
+  lingua-franca = buildVimPluginFrom2Nix {
```

Comment:
Please also commit the change in ``vim-plugin-names``.

Reply:
done